dataplane/config/config.go

```diff
@@ -121,6 +121,74 @@ func configRead(c *cli.Context) error {
 	return nil
 }
 
+// CheckConfigAndCommandFlagsDP : CAAS does not require workspace check
+func CheckConfigAndCommandFlagsDP(c *cli.Context) error {
+	err := fl.CheckRequiredFlagsAndArguments(c)
+	if err == nil {
+		return configReadDP(c)
+	}
+	return err
+}
+
+func configReadDP(c *cli.Context) error {
+	args := c.Args()
+	if args.Present() {
+		name := args.First()
+		if k := c.App.Command(name); k != nil {
+			// this is a sub-command invocation
+			return nil
+		}
+	}
+
+	server := c.String(fl.FlServerOptional.Name)
+	output := c.String(fl.FlOutputOptional.Name)
+	profile := c.String(fl.FlProfileOptional.Name)
+	refreshToken := c.String(fl.FlRefreshTokenOptional.Name)
+
+	if len(profile) == 0 {
+		profile = "default"
+	}
+
+	config, err := ReadConfig(GetHomeDirectory(), profile)
+	if err != nil {
+		utils.LogErrorAndExit(err)
+	}
+
+	set := func(name, value string) {
+		if err = c.Set(name, value); err != nil {
+			log.Debug(err)
+		}
+	}
+
+	if len(output) == 0 {
+		set(fl.FlOutputOptional.Name, config.Output)
+	}
+	PrintConfig(*config)
+	if len(server) == 0 {
+		set(fl.FlServerOptional.Name, config.Server)
+	}
+	if len(refreshToken) == 0 {
+		if len(config.RefreshToken) == 0 {
+			token := caasauth.NewRefreshToken(c.String(fl.FlServerOptional.Name))
+			err = WriteConfigToFile(GetHomeDirectory(), config.Server, config.Output, profile, config.Workspace, token)
```

Comment:
Do we need `config.Workspace` here?

Reply:
we don't need it. I will make the change